### DISCUSSION TITLE
Translation option

### DISCUSSION THREAD

**Discussion by Zerotask** [community/community#19477] [Mobile]
In the mobile app there is no translation option for discussions like in the web version which makes it harder to help others

**Comment by candyho** [comment]
Hey, thanks for your feedback! We will add this to our backlog to keep track of similar request.

**Reply by TPS** [reply]
Integrating Bing's autotranslate would fix this

**Reply by ByronScottJones** [reply]
It's been 4 YEARS since translation was suggested for the mobile app. This is way, way, overdue.

**Comment by lierenate2807** [comment]
hey.
takk for et virkelig viktig emne og føre opp her.
uten noe oversettelse her,blir det et stort hull som vi ikke kommer til og klare og fylle.
jeg mener vi burde absolutt ikke 
kopiere andres apper for oversettelse.
kunne vi ikke klart og lage vår egen "oversettelses-kart"?:) 
noe jeg vet vi klarer😀

**Reply by TPS** [reply]
Don't need to copy: GitHub, because also owned by Microsoft, has some Bing integration already, so they just need to enable its translation abilities, also.
